tests/snuba/api/endpoints/test_organization_events_stats_span_indexed.py

```diff
@@ -1581,3 +1581,94 @@ def test_time_spent_percentage_timeseries_fails(self):
             "Time_Spent_Percentage Is Not Enabled For This Request. Reason: Not Supported For Timeseries Requests"
             in response.data["detail"].title()
         )
+
+    def test_module_alias(self):
```

Comment:
Can you test this on the project alias as well?

Reply:
There's several project related tests, i can update them to pass